internal/loader/configuration_setting_loader.go

```diff
@@ -535,8 +560,17 @@ func (csl *ConfigurationSettingLoader) ExecuteFailoverPolicy(ctx context.Context
 func updateClientBackoffStatus(clientWrapper *ConfigurationClientWrapper, successful bool) {
 	if successful {
 		clientWrapper.BackOffEndTime = metav1.Time{}
-		clientWrapper.FailedAttempts = 0
+		// Reset FailedAttempts when client succeeded
+		if clientWrapper.FailedAttempts > 0 {
+			clientWrapper.FailedAttempts = 0
+		}
+		// Use negative value to indicate that successful attempt
+		clientWrapper.FailedAttempts--
```

Comment:
Set `clientWrapper.FailedAttempts = -1`  directly?

Reply:
Why not use `0` to indicate success?

Reply:
`0` can also represent the client is not used. negative value can make sure the client's status update when it succeeded.